server.go

```diff
@@ -705,6 +704,25 @@ func hasServices(advertised, desired wire.ServiceFlag) bool {
 	return advertised&desired == desired
 }
 
+// isSupportedNetAddressTypeV1 returns whether the provided address manager
+// network address type is supported by the addr wire message.
+func isSupportedNetAddressTypeV1(netAddressType addrmgr.NetAddressType) bool {
+	switch netAddressType {
+	case addrmgr.IPv4Address:
+	case addrmgr.IPv6Address:
+	case addrmgr.TORv2Address:
+		return true
+	}
+	return false
+}
+
+// getNetAddressTypeFilter returns a function that determines whether a
+// specific address manager network address type is supported by the
+// provided protocol version.
+func getNetAddressTypeFilter(pver uint32) addrmgr.NetAddressTypeFilter {
```

Comment:
Per effective Go, funcs really shouldn't be named with `get` as it is either redundant or not really descriptive of its intention.  In this case, it appears the intention is to return a filter for the supported address types based on the protocol version.  So, I'd suggest something like `supportedAddrTypesFilter(pver uint32)`.

As an aside, the comment doesn't match what it's really doing either.  It claims that it determines whether a specific address type is supported, but what it's really doing is returning a filter for the address types supported by the provided protocol version.